CONTRIBUTING.md

```diff
@@ -0,0 +1,246 @@
+# Contributing to our Blog
+
+Authors who are informed and passionate about technology and IT write technical
+blogs that range from high-level, conceptual overviews to deep dives with
+code snippets. They all offer insight into solving problems and getting the most
+out of applications and platforms. Many also add a personal perspective to the
+conversation. These authors share their journeys, trials, and triumphs.
+
+First, thanks for your interest in contributing and helping us craft quality
+content for our official Technical Blog. Second, in order to make contributing
+a pleasant experience while maintaining a visual and consistent content standard,
+complete these steps before writing and submitting a blog entry for publishing.
+
+For ideas of topics to blog about, check out the end of this file!
+
+#### Preparing for your blog entry
+
+1. Fork this repo (https://github.com/rackerlabs/technical-blog), then
+   clone your fork.
+
+2. Before you start working, make sure your local content is up-to-date and
+   merged with the right upstream branch:
+
+      ```bash
+         git remote add upstream git@github.com:rackerlabs/docs-developer-blog.git
```

Comment:
guess the upstream link should change to point to the technical-blog repo.

Reply:
Agreed